mods/tsunamissb/statuses.js

```diff
@@ -97,10 +97,10 @@ exports.BattleStatuses = {
 	tsunamiprince: {
 		exists: true,
 		onStart: function () {
-			this.add('c', '~Tsunami Prince', 'You think i came for the battle, when it\'s really your soul i want.');
+			this.add('c', '~Tsunami Prince', 'You think I came for the battle, when it\'s really your soul i want.');
```

Comment:
By the by, ever noticed that one with HH? XD that's on SG too. ~~ik you have touched their file more than I~~